docs/code-of-conduct-response.rst

```diff
@@ -1,76 +1,121 @@
+
+Reviewing the report
+~~~~~~~~~~~~~~~~~~~~
+
+Based on the report and any other available information, the response team will meet to determine, to the best of their ability:
+
+* What happened
+* Whether this event constitutes a Code of Conduct violation
+* Who, if anyone, was the bad actor(s)
+* What the appropriate resolution is
+
+A written record will be kept of each incident and its review. Although that sometimes seems superflous at the time, hectic environments like conferences can make it easy to forget or confuse details later.
 
-In addition, hotel/venue security and local authorities should be contacted when appropriate. Please contact one of the representatives listed in the conference brochure or posted on signs in the conference venues.
+As a result of the meeting, the response team may come to a conclusion about a resolution, or may conclude that additional information should be obtained. In the latter case, this additional information will be collected as soon as possible. Any conversations as part of this should generally be done by two people from the response team.
 
-CoC violations in community spaces
-----------------------------------
+Conflicts of interest
+~~~~~~~~~~~~~~~~~~~~~
 
-In case someone reports a code of conduct violation outside conferences or meetups, in one of the Write the Docs online spaces, or otherwise by email,
+As soon as reasonable, but at the latest at the report review meeting, team members should declare any conflicts of interest. This can mean being friends with one of the involved parties, or anything else that may make it harder to remain neutral.
 
-Initial receipt
-~~~~~~~~~~~~~~~
+A conflict of interest does not inherently mean the team member can no longer participate in the process, as that would make it very hard for the team to act on reports involving well-known people in the community. However, if a report concerns someone a team member is very close to, they probably should be not take part in the process. The team will decide together on where to draw this line in individual cases. Where possible, any conversations with offenders should not be done by people who know them, as it can be very unpleasant for everyone involved.
 
-Reports must be acknowledged as received within 24 hours or sooner.
+Resolutions
+~~~~~~~~~~~
 
-The staff must to review the incident and determine the following:
+The most common resolutions the response team can decide on are:
 
-* What happened.
-* Whether this event constitutes a code of conduct violation.
-* Who the bad actor was.
-* Whether this is an ongoing situation
-* If there is a threat to anyone's physical safety.
+* No action (if the team determines no violation occurred).
+* A private or pubic reprimand.
+* Demanding that a public apology is made.
```

Comment:
demanding > requiring (sounds less aggressive and more formal) [change everywhere on the list?]